osd_planning/aws-ccs.adoc

```diff
@@ -21,5 +21,6 @@ include::modules/osd-aws-privatelink-firewall-prerequisites.adoc[leveloffset=+2]
 .Additional resources
 
 *  xref:../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring]
+* xref:../osd_planning/aws-ccs.adoc#osd-security-groups-custom_aws-ccs[Security groups]
```

Comment:
This link opens the "Additional custom security" groups section. Is that the intended behavior, or should it point to the "Security groups" section?

Reply:
It should point to security groups and not the firewall module so slotted it in between the 2 modules in the assembly